src/components/Disputes/PrecedenceCampaign/PrecedenceCampaignModal.js

```diff
@@ -0,0 +1,119 @@
+import React from 'react'
+import { Button, GU, Modal, textStyle, useTheme, useViewport } from '@aragon/ui'
+
+import pcModalSvg from './assets/PCModal.svg'
+
+function PrecedenceCampaignModal({ opened, onClose }) {
+  const { width, height } = useViewport()
+  const theme = useTheme()
+
+  const verticalMode = width < 1000
+  const modalHeight = verticalMode
+    ? height - 40
+    : Math.max(700, Math.min(620, height - 40))
+
+  return (
+    <div>
+      <Modal
+        closeButton={false}
+        padding={0}
+        visible={opened}
+        width={Math.min(1055, width - 40)}
+        css={`
+          z-index: 4;
```

Comment:
I'm not sure, but do we need this z-index? Modal should already be rendering in a portal that's of higher z-index precedence than the main view via aragonUI.

Reply:
Yeah the problem is that it would render relative to the inner page but form your comment i'm realizing that we probably want to set that `z-index` at the  `<Root />` component

Reply:
Well actually if we do that, we have conflicts with the SideBar and Header's shadows

Reply:
Ah yes we have a new `Root.Provider` defined [on the content area](https://github.com/aragon/court-dashboard/blob/development/src/components/MainView.js#L65), so we need to escape it.

It would be nice to have named `Root.Provider` instances so we could choose :-)